storage-access.bs

```diff
@@ -162,7 +162,23 @@ When invoked on {{Document}} |doc|, the <dfn export method for=Document><code>ha
 
     ISSUE: "same authority" here is a placeholder for a future concept that allows user agents to perform [=same site=] checks while adhering to additional security aspects such as the presence of a cross-site parent document, see [whatwg/storage#142](https://github.com/whatwg/storage/issues/142#issuecomment-1122147159). In practice, this might involve comparing the [=site for cookies=] or performing a [=same site=] check with the top-level document.
 
-1. [=Queue a global task=] on the [=permissions task source=] given |global| to [=/resolve=] |p| with |global|'s [=environment/has storage access=].
+1. Let |topLevelSite| be the result of [=obtain a site|obtaining a site=] from the [=top-level origin=] of |doc|'s [=relevant settings object=].
+1. Let |embeddedSite| be the result of [=obtain a site|obtaining a site=] from |doc|'s [=Document/origin=].
+1. Run the following steps [=in parallel=]:
+    1. Let |Whether the User Agent Explicitly Allows Unpartitioned Cookie Access| be an algorithm, given a [=tuple=] |tuple| consisting of [=sites=], runs the following steps. This algorithm returns "none", "allow" or "disallow".
+        
+        Note: The "explicit allow" can come from user agent's per-site allow-lists or the user changing global browser settings, etc..
```

Comment:
I think I'd phrase it like this:

```suggestion
        Note: A user agent's settings might explicitly allow or disallow unpartitioned cookie access through per-site allow-lists, the user changing global browser settings, or similar custom overrides.
```

I also wonder if we need to explicitly mention that this is implementation-defined, but OTOH this note is non-normative anyway, and the text also implies it.